Source/Applications/SystemCenterNotification/Controllers/EmailTypeController.cs

```diff
@@ -240,9 +226,13 @@ void ConfigureRequest(HttpRequestMessage request)
                     .GetAwaiter()
                     .GetResult();
 
-                IEnumerable<DataSourceResponse> results = JsonConvert
-                    .DeserializeObject<IEnumerable<DataSourceResponse>>(responseJSON)
-                    .Select(r => new DataSourceResponseTSX(r));
+                JArray results = JArray.Parse(responseJSON);
+
+                for(int index = 0; index < results.Count(); index++)
+                {
+                    XElement data = JsonConvert.DeserializeObject<XElement>(results[index]["Data"].ToString());
+                    results[index]["Data"] = data.ToString() ?? "";
```

Comment:
Do you need all those other fields that came with the `DataSourceResponse` base class, like `Success`, `Created`, `Exception`, and `Model`?